its/src/test/java/com/sonar/maven/it/suite/MavenTest.java

```diff
@@ -167,8 +167,8 @@ void shouldSupportJeeProjects() {
     ORCHESTRATOR.executeBuild(build);
 
     // src/main/webapp is analyzed by web and xml plugin
-    // scanning all files, so 2 more files (ejb-module/src/main/resources/META-INF/ejb-jar.xml and web-module/src/main/webapp/index.jsp)
-    assertThat(getMeasureAsInteger("com.sonarsource.it.samples.jee:parent", "files")).isEqualTo(10);
+    // including resources, so one more file (ejb-module/src/main/resources/META-INF/ejb-jar.xml)
```

Comment:
Let's remove this line because files inside of src/main/resources are not picked up
```suggestion
```